resources/prosody-plugins/mod_muc_lobby_rooms.lua

```diff
@@ -158,6 +158,20 @@ function filter_stanza(stanza)
         elseif stanza.name == 'iq' and stanza:get_child('query', DISCO_INFO_NS) then
             -- allow disco info from the lobby component
             return stanza;
+        elseif stanza.name == 'message' then
```

Comment:
Hi @damencho we currently allow private messages and also messages only  to a moderator from line 163 to 173. (Or should we implement it differently). I think we are left with checking messages from a moderator

Reply:
Let me check again, when I was looking at it I had the impression that lobby users can send messages to lobby users privately or via group messages.

Reply:
So you allow all messages between lobby participants and group messages to moderators. So I was assuming group chats between participants are possible ... 

Can you easily check and do it: "messages to and from moderators are only allowed"... ?
It shouldn't be possible to find other participants ids, but just in case the check ^ seems better, I think.

Reply:
That makes sense, we will check on that